lib/main.coffee

```diff
@@ -195,13 +202,15 @@ module.exports =
         pep8warn = atom.config.get('linter-flake8.pep8ErrorsToWarnings')
         flakeerr = atom.config.get('linter-flake8.flakeErrors')
         projDir = @getProjDir(filePath) or path.dirname(filePath)
-        execPath = fs.normalize(@applySubstitutions(atom.config.get('linter-flake8.executablePath'), projDir))
+        execPath = fs.normalize(@applySubstitutions(atom.config.get('linter-flake8.executablePath'),
+          projDir, path.sep))
         cwd = path.dirname(textEditor.getPath())
-        return helpers.exec(execPath, parameters, {stdin: fileText, cwd: cwd}).then (result) ->
+        return helpers.exec(execPath, parameters, {stdin: fileText, cwd: cwd, stream: 'both'}).then (result) ->
+          console.log('flake8 stderr: ' + result.stderr)
```

Comment:
This should only log if there is contents in there, and you might want to wrap this in a check on `atom.inDevMode()`, depending on if we should _always_ be notifying the user, or only if they are looking for the information.